db.go

```diff
@@ -52,6 +52,25 @@ var migrations = &migrate.MemoryMigrationSource{
 				`,
 			},
 		},
+		{
+			Id: "3",
+			Up: []string{
+				`CREATE TABLE IF NOT EXISTS forwarding_history (
+                                        id INTEGER PRIMARY KEY,
+                                        add_time TIMESTAMP NOT NULL,
```

Comment:
So for incomplete records (after restart), this will just be the zero time? If it comes from unix nano, maybe it isn't very recognizable as such? Perhaps something similar can be said about some of the other fields.

Reply:
Yeah, all the more reason to not store these incomplete HTLCs. 

Updated to not store anything with a zero add time and just debug log: 
`Not storing incomplete htlc resumed after restart: chan (idx) -> chan (idk)`